src/CTA.WebForms2Blazor/Factories/FileConverterFactory.cs

```diff
@@ -13,6 +14,13 @@ public class FileConverterFactory
         private readonly WorkspaceManagerService _blazorWorkspaceManager;
         private readonly WorkspaceManagerService _webFormsWorkspaceManager;
         private readonly ClassConverterFactory _classConverterFactory;
+        
+        public readonly HashSet<string> StaticResourceExtensions = new HashSet<string>
```

Comment:
Its possible that extensions outside of this list ought to be placed in the wwwroot folder. For instance, the customer may have downloadable .csv, .pdf, or .docx files. As far as I know there isn't a surefire way to determine which files ought to be placed in the folder and which ones shouldn't, not sure if guessing like this or just tossing all files in wwwroot is the right approach though

Reply:
Yeah this isn't the best way to do it, but at least its good enough for demoing for eShopOnBlazor. In the future we might need to add a note for the customer to make sure and check if the correct files are in wwwroot

Reply:
True, or using some kind of optional blacklist configured by the customer before the run could be used, but this works for now

Reply:
I think we are ok with this for now. For scope of project.